docs/index.md

```diff
@@ -17,7 +17,7 @@ id: home
     <div class="marketing-col">
       <h3>Virtual DOM</h3>
       <p>
-        React abstracts away the DOM from you, giving a simpler programming model and better performance. React can also render on the server using Node, and it can power native apps using <a href="https://facebook.github.io/react-native/">React Native</a>.
+        React abstracts the DOM away from you, giving a simpler programming model and better performance. React can also render on the server using Node, and it can power native apps using <a href="https://facebook.github.io/react-native/">React Native</a>.
```

Comment:
I feel like the "from you" is what makes this sentence awkward.  With the "from you", both sentences read awkwardly.  Without the "from you", both versions seem fine.  Maybe the correct fix here is to remove the "from you" rather than moving the "away".

Thoughts @zpao?